src/Tools/GenerateProps/targetgroups.props

```diff
@@ -107,7 +107,7 @@
       <TargetingPackNugetPackageId>Microsoft.TargetingPack.NETFramework.v4.6.3</TargetingPackNugetPackageId>
       <NuGetTargetMoniker>.NETFramework,Version=v4.6.3</NuGetTargetMoniker>
       <Imports>net463</Imports>
-      <CompatibleWith>netstandard2.0</CompatibleWith>
+      <CompatibleWith>netstandard</CompatibleWith>
```

Comment:
I still don't like that we need to "remember" to do this.  I think I want to add the equivalent moniker feature.